src/liboslexec/batched_llvm_gen.cpp

```diff
@@ -1303,23 +1312,35 @@ LLVMGEN (llvm_gen_mul)
 
     // multiplication involving closures
     if (Result.typespec().is_closure()) {
-        OSL_ASSERT(0 && "incomplete");
-        llvm::Value *valargs[3];
+
+        BatchedBackendLLVM::TempScope temp_scope(rop);
+
+        llvm::Value *valargs[5];
         valargs[0] = rop.sg_void_ptr();
+        valargs[1] = rop.llvm_void_ptr(Result, 0);
```

Comment:
Think the 2nd parameter will default to 0 automatically, so
`valargs[1] = rop.llvm_void_ptr(Result);`
would do.

Reply:
Done.